src/sentry/api/endpoints/useravatar.py

```diff
@@ -1,5 +1,7 @@
 from __future__ import absolute_import
 
+import uuid
```

Comment:
Can you do just `from uuid import uuid4`? Attribute lookups in python aren't free, so this is technically faster and better practice for python.